sdk/storage/storage-blob/test/perfstress/track-2/listBlobs.spec.ts

```diff
@@ -0,0 +1,38 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import { PerfStressOptionDictionary } from "@azure/test-utils-perfstress";
+
+// Expects the .env file at the same level as the "test" folder
+import * as dotenv from "dotenv";
+import { StorageBlobTest } from "./storageTest.spec";
+dotenv.config();
+
+interface StorageBlobListTestOptions {
+  count: number;
+}
+
+export class StorageBlobListTest extends StorageBlobTest<StorageBlobListTestOptions> {
+  public options: PerfStressOptionDictionary<StorageBlobListTestOptions> = {
+    count: {
+      required: true,
+      description: "Number of blobs to be listed",
+      longName: "count",
+      defaultValue: 10
+    }
+  };
+
+  public async globalSetup() {
+    await super.globalSetup();
+    const tasks = [];
+    for (let i = 0; i < this.parsedOptions.count.value!; i++) {
+      tasks.push(this.containerClient.uploadBlockBlob(`blob-${i}`, Buffer.alloc(0), 0));
+    }
+    await Promise.all(tasks);
+  }
+
+  async runAsync(): Promise<void> {
+    for await (const _ of this.containerClient.listBlobsFlat()) {
```

Comment:
Does this enumerate each blob listed, to ensure we are consuming the results?

Reply:
Yes, it does enumerate.

Hoewever, moved to 
```
    for await (const segmentResponse of this.containerClient.listBlobsFlat().byPage()) {
      for (const _ of segmentResponse.segment.blobItems) {
      }
    }
```
...to make it look similar to track 1.